frontend/class-opengraph-image.php

```diff
@@ -564,13 +564,13 @@ protected function is_valid_image_url( $url ) {
 	 * @return string The path of the image URL. Returns an empty string if URL parsing fails.
 	 */
 	protected function get_image_url_path( $url ) {
-		$parsed_url = wp_parse_url( $url );
+		$parsed_url = wp_parse_url( $url, PHP_URL_PATH );
 
-		if ( $parsed_url === false ) {
+		if ( empty($parsed_url) ) {
 			return '';
 		}
 
-		return $parsed_url['path'];
+		return $parsed_url;
 	}
```

Comment:
It's possible to replace it with ( `(string) null === ''`):
```php
protected function get_image_url_path( $url ) {
	return (string) wp_parse_url( $url, PHP_URL_PATH );
}
```
It's little faster. @andizer What you think?

Reply:
@andizer should i implement that?